kitty/css/style.css

```diff
@@ -0,0 +1,530 @@
+  }
+  .icon-map:before {
+  content: "\e904";
+  }
+  .icon-camera:before {
+  content: "\e905";
+  }
+  .icon-creditcard:before {
+  content: "\e906";
+  }
+  .icon-auditlog:before {
+  content: "\e907";
+  }
+  .icon-checkmark:before {
+  content: "\e908";
+  }
+  .icon-gallery:before {
+  content: "\e909";
+  }
+  .icon-notification:before {
+  content: "\e910";
+  }
+  .icon-sort-ascending:before {
+  content: "\e911";
+  }
+  .icon-sort-descending:before {
+  content: "\e912";
+  }
+  .icon-switch:before {
+  content: "\e913";
+  }
+  .icon-upload:before {
+  content: "\e943";
+  }
+  .icon-chevron-thick-both:before {
+  content: "\e915";
+  }
+  .icon-emptystate:before {
+  content: "\e90a";
+  }
+  .icon-unlock:before {
+  content: "\e90b";
+  }
+  .icon-vacation:before {
+  content: "\e90c";
+  }
+  .icon-audit-arrow:before {
+  content: "\e90d";
+  }
+  .icon-timer:before {
+  content: "\e90e";
+  }
+  .icon-disable:before {
+  content: "\e90f";
+  }
+  .icon-pointer-down:before {
+  content: "\e914";
+  }
+  .icon-pointer-up:before {
+  content: "\e916";
+  }
+  .icon-pointer-left:before {
+  content: "\e917";
+  }
+  .icon-pointer-right:before {
+  content: "\e918";
+  }
+  .icon-csv:before {
+  content: "\e919";
+  }
+  .icon-zip:before {
+  content: "\e91a";
+  }
+  .icon-attention-sym:before {
+  content: "\e91c";
+  }
+  .icon-question-sym:before {
+  content: "\e91d";
+  }
+  .icon-star-solid:before {
+  content: "\e91e";
+  }
+  .icon-pause:before {
+  content: "\e91b";
+  }
+  .icon-openbox:before {
+  content: "\e91f";
+  }
+  .icon-resume:before {
+  content: "\e920";
+  }
+
+
+body{
+    background-color: #f1f1f4;
+    font-size:1em;
+}
+
+header .row{
+  margin:0;
+  padding:0;
+  background-color: #03BA5B;
+  padding:10px 0 0 20px;;
+  height:50px;
+  width:100%;
+  position:fixed;
+  z-index: 10;
+  box-shadow: 0px 0px 5px #414141;
+  font-size:12px;
+}
+
+
+.row{
+  margin:0px 0 0 200px;
+  padding-top:74px;
+  
+}
+
+.col h1{
+  font-size:32px;
+  line-height:32px;
+    text-transform: capitalize;
+
+}
+
+.col p{
+ font-size:12px;
+
+}
+
+.card{
+  margin-bottom:12px;
+}
+
+
+/* Navigation */
+nav{
+    background-color: #4B555B;
+    color:white;
+    text-transform: uppercase;
+    font-size:12px;
+    width:200px;
+    min-height: 100vh;
+    float:left;
+    margin-top:50px;
+    position:fixed;
+    z-index:9;
+}
+
+nav span{
+    padding:10px;
+    font-size:16px;
+  
+}
+nav a{
+    color:white;
+    text-decoration: none;
+}
+
+nav li:hover{
+    background-color:#343B3F;
+}
+
+nav li{
+   padding:20px;
+   
+ 
+}
+
+/* Top Nav */
+
+.profile{
+margin-top:-10px;
+margin-right:20px;
+float:right;
+height:50px;
+padding:19px 0 19px 0;
+font-size:12px;
+color:white;
+}
+
+.profile-comment{
+    margin-top:-10px;
+    margin-right:20px;
+    float:left;
+    height:50px;
+    padding:19px 0 19px 0;
+    font-size:12px;
+    color:white;
+    }
+
+.profile-photo{
+ 
+    margin:-19px 0 0 0px; 
+    }
```

Comment:
Watch the indentation here ☝️ , its important that we keep consistent tabbing especially when your app gets larger. You will thank yourself!